content/millennium/r4/individuals/person.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Person | R4 API
+---
+
+# Person
+
+* TOC
+{:toc}
+
+## Overview
+
+The Person resource identifies an individual outside of a specific health care context providing a mechanism to link person resources across different facilities or organizations.
+
+The following fields are returned if valued:
+
+* [Person id](http://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Identifier (Cerner alias Federated Person Principal)](http://hl7.org/fhir/R4/person-definitions.html#Person.identifier){:target="_blank"}
+* [Name](http://hl7.org/fhir/R4/person-definitions.html#Person.name){:target="_blank"}
+* [Telecom](http://hl7.org/fhir/R4/person-definitions.html#Person.telecom){:target="_blank"}
+* [Gender](http://hl7.org/fhir/R4/person-definitions.html#Person.gender){:target="_blank"}
+* [Date of birth](http://hl7.org/fhir/R4/person-definitions.html#Person.birthDate){:target="_blank"}
+* [Address](http://hl7.org/fhir/R4/person-definitions.html#Person.address){:target="_blank"}
+* [Active](http://hl7.org/fhir/R4/person-definitions.html#Person.active){:target="_blank"}
```

Comment:
We should we secure links here.

Reply:
Changed in commit 8f913dce83505df0c6e3815d58fb4cf8367d8b44

Reply:
+1